jekyll/_cci2/api-intro.md

```diff
@@ -34,7 +34,7 @@ The current categories of the API v2 endpoints are:
 
 **Note:** Portions of the CircleCI API v2 remain under "Preview". Preview endpoints are not yet fully supported or considered generally available. Breaking changes to API v2 Preview endpoints are planned in advance and are announced in the [API v2 breaking changes log](https://github.com/CircleCI-Public/api-preview-docs/blob/master/docs/breaking.md).
 
-## Getting Started with the API v2
+## Getting started with the API v2
```

Comment:
Oh how weird, it didn't affect API here!

Reply:
I think that was one of the ones I manually changed and updated.